### PR TITLE
fixes for couple of issues in the UI

### PR DESCRIPTION
**Pull request by madhuv-msft**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azurepipelines/azdevops-kube-summary",
-  "version": "2.2.2",
+  "version": "2.2.4",
```
Comment (madhuv-msft) [inline package.json]: do we need this change?

File: package.json [hunk 2]
```diff
@@ -76,6 +76,6 @@
   "jest-junit": {
     "suiteName": "L0 Uts",
     "outputDirectory": ".",
-    "outputName": "./jest-l0-uts.xml"
+    "outputName": "./dist_tests/jest-l0-uts.xml"
```
Comment (madhuv-msft) [inline package.json]: test changes did to have a clean slate

File: src/WebUI/Components/PodsComponent.tsx [hunk 3]
```diff
@@ -112,7 +113,11 @@ export class PodsComponent extends BaseComponent<IPodsComponentProperties> {
                     let customDescription: React.ReactNode = null;
 
                     if(pod.status.message) {
-                        customDescription = <Tooltip showOnFocus={true} text={pod.status.message}>{pod.status.reason}</Tooltip>;
+                        customDescription = (
+                            <Tooltip showOnFocus={true} text={pod.status.message}>
+                                <div className="kube-status-desc">{pod.status.reason}</div>
```
Comment (madhuv-msft) [inline src/WebUI/Components/PodsComponent.tsx]: this is fix one.

File: src/WebUI/Components/PodsComponent.tsx [hunk 4]
```diff
@@ -125,9 +130,7 @@ export class PodsComponent extends BaseComponent<IPodsComponentProperties> {
                     />);
 
             case podAgeKey:
-                return(
-                    <Ago date={new Date(pod.status.startTime)} />
-                );
+                return (pod.status && pod.status.startTime ? <Ago date={new Date(pod.status.startTime)} /> : null);
```
Comment (madhuv-msft) [inline src/WebUI/Components/PodsComponent.tsx]: this is fix 2

File: src/WebUI/Components/PodsComponent.tsx [hunk 5]
```diff
@@ -12,6 +12,7 @@ import { Utils } from "../Utils";
 import { StatusSize, Status } from "azure-devops-ui/Status";
 import { Tooltip } from "azure-devops-ui/TooltipEx";
 import { PodStatusComponent } from "./PodStatusComponent";
+import "azure-devops-ui/Components/TooltipEx/Tooltip.scss";
```
Comment (madhuv-msft) [inline src/WebUI/Components/PodsComponent.tsx]: this is not included in the original devops, i will open an issue with our devops repo.

File: src/WebUI/Components/PodsComponent.tsx (hunk 3 above)
Comment (savenkat) [inline src/WebUI/Components/PodsComponent.tsx]: change to span if you don't intend to have any more children of this element

**Pull request merged**